content/core-concepts/metrics.md

```diff
@@ -53,10 +53,10 @@ they want to collect.
 
 In order to aggregate measurements and export, users need to define views.
 A view allows recorded measurements to be aggregated with a one of the
-aggregation methods set by the user cumulatively.
-All recorded measurements is broken down by user-provided [tag](/core-concepts/tags) keys.  
+aggregation methods set by the user cumulatively, as well as [tags](/core-concepts/tags) to disambiguate
```

Comment:
I feel the original phrase "break down metrics by dimensions" is clearer to me. "Disambiguate" here implies that a view with no tag key is ambiguous, which I don't think it's the case.

Reply:
Roger that, thanks I'll revert this.